lib/perl/Genome/InstrumentData/AlignmentResult/Merged.t

```diff
@@ -19,7 +18,17 @@ BEGIN {
 use above 'Genome';
 use Genome::Test::Factory::SoftwareResult::User;
 
-use_ok('Genome::InstrumentData::AlignmentResult::Merged');
+my $pkg = 'Genome::InstrumentData::AlignmentResult::Merged';
+use_ok($pkg);
+
+# Override methods for testing. We do this here so we don't worry about UR_DBI_NO_COMMIT in actual code.
```

Comment:
I don't think we can safely disregard `UR_DBI_NO_COMMIT` in the code.  Do we want the callback to fire and remove the per-lane BAM if it is turned on and the merged BAM we are creating won't be saved?